2199/2/versus.rb

```diff
@@ -0,0 +1,141 @@
+require 'yaml'
+require 'bundler'
+Bundler.require
+
+# :reek:InstanceVariableAssumption
+# class Battle to collect information
+class Battle
+  def initialize(filename)
+    @filename = filename
+  end
+
+  def bad_words_count
+    words.count do |word|
+      word.include?('*') || RussianObscenity.obscene?(word)
+    end
+  end
+
+  def words_count
+    words.count
+  end
+
+  def rounds_count
+    rounds_count = text.scan(/Раунд \d/).length
```

Comment:
This is fine but I just don't like temporary variables. So let's remove this. Instead of placing this to a temporary variable you can add a method like `#round_word_in_text_count` or something like this.